app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailController.scala

```diff
@@ -1,113 +0,0 @@
-
-object CitizenDetailsController extends BaseController with Logging {
-
-  type NINO = String
-
-  case class Person(firstName:   Option[String],
-                    lastName:    Option[String],
-                    dateOfBirth: Option[LocalDate])
-
-  case class Address(line1:    Option[String],
-                     line2:    Option[String],
-                     line3:    Option[String],
-                     line4:    Option[String],
-                     line5:    Option[String],
-                     postcode: Option[String],
-                     country:  Option[String]
-  )
-
-  case class Response(person: Option[Person], address: Option[Address])
-
-  implicit val personWrites: Writes[Person] = Json.writes[Person]
-  implicit val addressWrites: Writes[Address] = Json.writes[Address]
-  implicit val responseWrites: Writes[Response] = Json.writes[Response]
-
-  val responseGen: Gen[Response] = {
-    import Gen._
-
-    val personGen = for {
-      fnameO ← Gen.some(Gen.forename())
-      snameO ← Gen.some(Gen.surname)
-      dobO ← Gen.some(Gen.date(1940, 2017))
-
-    } yield Person(fnameO, snameO, dobO)
-
-    val addressGen = for {
-      address ← Gen.ukAddress
-      postcode ← some(Gen.postcode)
-      country ← some(const("GB"))
-    } yield {
-      val (l1, l2, l3, l4, l5) = address match {
-        case Nil                     ⇒ (None, None, None, None, None)
-        case a :: Nil                ⇒ (Some(a), None, None, None, None)
-        case a :: b :: Nil           ⇒ (Some(a), Some(b), None, None, None)
-        case a :: b :: c :: Nil      ⇒ (Some(a), Some(b), Some(c), None, None)
-        case a :: b :: c :: d :: Nil ⇒ (Some(a), Some(b), Some(c), Some(d), None)
-        case a :: b :: c :: d :: e   ⇒ (Some(a), Some(b), Some(c), Some(d), Some(e.mkString(", ")))
-      }
-      Address(l1, l2, l3, l4, l5, postcode, country)
-    }
-
-    for {
-      personO ← some(personGen)
-      addressO ← some(addressGen)
-    } yield Response(personO, addressO)
-  }
-
-  def retrieveDetails(nino: NINO) = Action { implicit request ⇒
-    DummyData.find(nino).map(toResponse).orElse(responseGen.seeded(nino)).map { response ⇒
-      logger.info(s"[CitizenDetailsController] Responding to request from $nino with details $response")
-      Ok(Json.toJson(response))
-    }.getOrElse{
-      logger.warn(s"[CitizenDetailsController] Could not find or generate data for nino $nino")
-      InternalServerError
-    }
-  }
-
-  private def toResponse(userInfo: UserInfo): Response = Response(
-    Some(Person(
-      Some(userInfo.forename),
-      userInfo.surname,
-      userInfo.dateOfBirth)),
-    Some(Address(
-      userInfo.address.line1,
-      userInfo.address.line2,
-      userInfo.address.line3,
-      userInfo.address.line4,
-      userInfo.address.line5,
-      userInfo.address.postcode,
-      userInfo.address.country
-    ))
-  )
-
-  def encode(nino: String): String = new String(Base64.getEncoder.encode(nino.getBytes()))
-
-}
```

Comment:
I've noted there are changes under 2 tickets here, but I've discussed with Andy and I'll merge on this occasion, as the code removed is dead.